app/helpers/multiple_metadata_fields_helper.rb

```diff
@@ -1,4 +1,35 @@
 module MultipleMetadataFieldsHelper
+  
+  #resoure_type data are ["ConferenceItem default Conference paper (unpublished)"] and ["Image default Still image"] %>
+  def readable_resource_type(data)
```

Comment:
I can push this code now for your review? It works, it's only a change of 4/5 lines of code.

Reply:
Yep if you want to, thanks.

 However, since your brought up the human_readable_resource_type helper,  it got me thinking on what underlying problem that method solves and if we addressing that problem at the right place. 

The main issue with the resource_type field is that it stores the wrong data beyond what the user selected. For instance, if a user selects 'Still image' from the dropdown if will save "Image default Still image". You can see that both are not thesame. In a similar vein if "Conference paper (unpublished)" is selected from the UI it saves "ConferenceItem default Conference paper (unpublished)". You can confirm that by querying the DB or inspecting it on the frontend and you will see something similar to what is pasted below.

     ["ConferenceItem default Conference paper (unpublished)"] and ["Image default Still image"] 

There are other dropdowns in Hyku like Institution meta-field which stores data exactly as it is selected in the dropdown and for those we don't need  any helper methods.
So that means human_readable_resource_type is a merely a band aid that conceals the real problem which is inaccurate storing of selected data.

So we can add the code needed to merge the two methods but I will suggest we address the underlying problem of saving more data than the user selected. That is why for creator, contributor and editor, I am also hoping to address the saving of default fields at the right layer.

Reply:
Ok I will push the code. 

The data saved for `resource_type` is a design decision I made to enable custom options for different types of works (i.e. Article, Dataset etc) in the submission form menu; and to pre-select a default option.
All other metadata drop-down menus offer the same options across all templates.
I don't see it as wrong data - rather it may become useful in the future to have the model and whether it is the default option or not.
However, design is open for discussion. Feel free to add this to the refactoring list.